react-native/ios/Keybase/AppDelegate.swift

```diff
@@ -33,9 +33,17 @@ class AppDelegate: UIResponder {
       let SecurityAccessGroupOverride = false
     #endif
 
+    var home = AppDefault.HomeDirectory.stringValue ?? ""
```

Comment:
Needed to fix this issue for ios. We can't make a folder ~/2, we can make ~/Library/2 though so this does that